packages/gatsby/src/schema/node-model.js

```diff
@@ -71,6 +71,21 @@ class LocalNodeModel {
     this._prepareNodesQueues = {}
     this._prepareNodesPromises = {}
     this._preparedNodesCache = new Map()
+    this.replaceTypeKeyValueCache()
+  }
+
+  /**
+   * Replace the cache either with the value passed on (mainly for tests) or
+   * an empty new Map.
+   *
+   * @param {undefined | Map<string, Map<string, Set<Node>> | Map<string, Node>>} map
+   *   (This cached is used in redux/nodes.js and caches a set of buckets (Sets)
+   *   of Nodes based on filter and tracks this for each set of types which are
+   *   actually queried. If the filter targets `id` directly, only one Node is
+   *   cached instead of a Set of Nodes.
+   */
+  replaceTypeKeyValueCache(map = new Map()) {
+    this._typedKeyValueIndexes = new Map() // See redux/nodes.js for usage
```

Comment:
is this a mistake not using the argument to assign?

Reply:
The ... Eh ... yes? :/